static/js/components/EmploymentTab.js

```diff
@@ -12,10 +12,14 @@ import moment from 'moment';
 
 import { generateNewWorkHistory } from '../util/util';
 import ProfileTab from "../util/ProfileTab";
-import { saveAndContinue } from '../util/profile_edit';
+import { saveProfileStep } from '../util/profile_edit';
 
 class EmploymentTab extends ProfileTab {
-  nextUrl = "/profile/privacy";
+  constructor(props) {
+    super(props);
+    this.prevUrl = "/profile/education";
+    this.nextUrl = "/profile/privacy";
```

Comment:
I prefer the instance property pattern to the constructor pattern, I think it's just a little cleaner. Could you switch this to:

```js
class EmploymentTab extends ProfileTab {
  nextUrl = "...";
  prevUrl = "...";
  ...
}
```